.gitlab-ci.yml

```diff
@@ -20,6 +21,7 @@ build-windows:
   tags:
     - windows
   script:
+    - set RUSTFLAGS=-C target-feature=+crt-static
```

Comment:
I think there's a way to consolidate flags between stages in GitLab. Might be a good option to keep it tidy.

Reply:
I've never used gitlab that much, especially CI. I just tweaked around the Rust stuff I know :P